ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -738,7 +766,13 @@ - (unsigned long) getUsedMemory {
 }
 
 - (NSDictionary *) getAvailableLocationProviders {
-#if !TARGET_OS_TV
+#if TARGET_OS_OSX
```

Comment:
Seems like this would be better restructured as a series instead of branching.
Initialization with locationServicesEnabled
if OSX or iOS add significantLocationChangeMonitorAvailable and headingAvailable
if iOS add isRangingAvailable